src/random.cpp

```diff
@@ -436,7 +464,23 @@ static void SeedSlow(CSHA512& hasher) noexcept
     SeedTimestamp(hasher);
 }
 
-static void SeedSleep(CSHA512& hasher)
+/** Extract entropy from rng, strengthen it, and feed it into hasher. */
+static void SeedStrengthen(CSHA512& hasher, RNGState& rng) noexcept
+{
+    // Once per minute, strengthen the hash for 10ms.
+    static std::atomic<int64_t> last_strengthen{0};
+    int64_t current_time = GetTimeMicros();
+    if (current_time > last_strengthen + 60000) {
+        // Generate 32 bytes of entropy from the RNG, and a copy of the entropy already in hasher.
+        unsigned char strengthen_seed[32];
+        rng.MixExtract(strengthen_seed, 32, CSHA512(hasher), false);
```

Comment:
nit: `sizeof(strengthen_seed)` would avoid the duplication here.

Reply:
Fixed.